README.md

```diff
@@ -39,3 +39,24 @@ When we took on the task of importing the standards, a few principles guided our
 * **Easy to fix**: Converting 50 state standards is a monumental task (and one we're grateful ASN takes on). There will be typos and we want users to find them and fix them. Thus, each set of standards is versioned and changes can be submitted for approval.
 
 * **Easy to add to**: Schools & districts create their own standards (outside of their state). They should be able to publish those standards and share them with their vendors.
+
+## Development
+
+There are a few tasks that must be completed to get running with a local copy of the project for development purposes:
+
+* Sign up with [Algolia](https://www.algolia.com) and obtain your Application ID and Admin API Key.
+* Set the Algolia keys in your env: `export ALGOLIA_APPLICATION_ID=[APPID]; export  ALGOLIA_API_KEY=[KEYID]` (to make this persistent, set in your bash profile)
+* Install [MongoDB](https://www.mongodb.org/)
+* Run `mongorestore sources/jurisdiction-mongodb-dump` to do an initial import of jurisdictions
```

Comment:
@aleksclark Ahh -- I understand the comment you left this morning. Let me know if the new dump I sent you works and, if so, feel free to add it. I'm guessing that we just need to rename the folder inside the `jurisdiction-mongodb-dump` to be `common-standards-project`, but I don't do enough with `mongodump` to know the specifics.